src/index.js

```diff
@@ -0,0 +1,74 @@
+import '../style/style.scss';
+
+const form = document.querySelector('form');
+
+const myCity = document.getElementById('city-input');
+const btnGetWeather = document.getElementById('get-weather');
+const result = document.getElementById('result');
+const resultInner = document.createElement('div');
+resultInner.id = 'inner-content';
+resultInner.classList.add('text-center', 'mt-2', 'text-white');
+const temp = document.createElement('span');
+const tempUnit = document.createElement('h6');
+const h3 = document.createElement('h3');
+const p = document.createElement('p');
+const p1 = document.createElement('p');
+const p2 = document.createElement('p');
+const p3 = document.createElement('p');
+temp.style.fontSize = '6rem';
+temp.style.position = 'relative';
+tempUnit.style.position = 'absolute';
+tempUnit.style.top = '15px';
+tempUnit.style.right = '-10px';
+
+let url;
+let city;
+
+function reset() {
+  form.reset();
+}
+
+function displayRemove() {
+  resultInner.innerHTML = '';
+}
+
+async function getData(url) {
```

Comment:
- [ ] [required] You should have used ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from Pathwright)
  - [ ] Usage of arrow functions instead of the function keyword